backend/services/createJWTSig.ts

```diff
@@ -0,0 +1,28 @@
+import { Buffer, createHmac, getNumericDate } from "../api/deps.ts";
+
+const base64UrlEncode = (data: string): string => {
+	return Buffer.from(data, "utf-8").toString("base64");
+};
+
+const headerObject = {
+	typ: "JWT",
+	alg: "HS256",
+};
+// Secret key for JWT
+const jwtKey = "59c4b48eac7e9ac37c046ba88964870d";
```

Comment:
Secret key should be in an environment file, not in the code. Please do not use this key anymore. It has already been compromised.

Reply:
(and the environment file should never be added to source control)

Reply:
Normally I'd never have the key in plain text (the above key is one used for testing to see the effects of creating JWT tokens). I created my own .env file with a JWT key, but I dont know of a way to share the secret key without having to put in a readme what the key is so everyone will have it.

Reply:
in prod, we will put this on a server with a key. for testing, you should add instructions in SETUP.md for generating a key and putting it in .env.local